src/typescript/Scala.tmLanguage.ts

```diff
@@ -404,7 +404,7 @@ export const scalaTmLanguage: TmLanguage = {
     'using': {
       patterns: [
         {
-          match: `\\(\\s*(using)`,
+          match: `(?<=\\()\\s*(using)\\s(?=\\s*(?:[\\w\\d\\(\\.\\(\\{'"]))`,
```

Comment:
Could this regex be simplified to the following?

```js
`(?<=\\()\\s*(using)\\s+(?=[\\w\\d\\(\\.\\(\\{'"])``
```

Reply:
Good catch. I will update it.